src/__tests__/lib/containers/entity/annotations/SchemaDrivenAnnotationsEditor.test.tsx

```diff
@@ -191,6 +195,17 @@ describe.skip('SchemaDrivenAnnotationEditor tests', () => {
         return res(ctx.status(201), ctx.json({ token: mockAsyncTokenId }))
       },
     ),
+    rest.get(
```

Comment:
A while ago we changed how SynapseClient requests the result for asynchronous jobs. This accomodates that change.